pkgs/by-name/ro/roon-tui/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, lib
+, rustPlatform
+, fetchFromGitHub
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "roon-tui";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "TheAppgineer";
+    repo = pname;
+    rev = version;
+    hash = "sha256-rwZPUa6NyKs+jz0+JQC0kSrw0T/EL+ms2m+AzHvrI7o=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "roon-api-0.1.1" = "sha256-aFcS8esfgMxzzhWLeynTRFp1FZj2z6aHIivU/5p+uec=";
+    };
+  };
+
+  meta = {
+    description = "A Roon Remote for the terminal";
+    homepage = "https://github.com/TheAppgineer/roon-tui";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ MichaelCDormann ];
+    platforms = [ "x86_64-linux" ];
```

Comment:
When you fix the platforms list this way, you are basically saying the upstream developers develop it for that specific platform only. A typical example is the `zsnes` emulator, written in x86 assembly.

I believe this is not the case here. Just remove this line:

```suggestion
```

Here around Nixpkgs we just wait for volunteers. Indeed a lot of people help keeping mpv and emacs on Darwin.

Until then, if ofborg fails then you can add something like

 ```nix
meta = { 
  . . . 
  broken = stdenv.isDarwin; # ofborg failed
}
```

Reply:
Got it, that makes sense